accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/exceptions/AccessionIsNotPendingException.java

```diff
@@ -17,6 +17,10 @@
  */
 package uk.ac.ebi.ampt2d.commons.accession.core.exceptions;
 
+/**
+ * Thrown when accession can not be generated for an object;
```

Comment:
This exception is thrown if there is an error while postSave of accessions to ContiguosBlockRepository to denote the accession has been used. If generatedAccessions doesn't match with the accessions to commit, this error is thrown.

Reply:
+1. Also `throw`.